execution/executor-types/src/lib.rs

```diff
@@ -83,6 +83,12 @@ pub trait ChunkExecutorTrait: Send + Sync {
     fn reset(&self) -> Result<()>;
 }
 
+pub struct SnapshotDelta {
```

Comment:
nit: `SnapshotDelta` is a very generic term and doesn't signfy what delta are we talking about. May be `StateSnapshotDelta` or `StateDelta` is a better name?

Reply:
changed to state_snapshot_delta